site/content/api/Command-And-Control-API.md

```diff
@@ -57,22 +53,23 @@ After execution of a control message on a device, device MUST send a response me
 
 **Preconditions**
 
-1. Client has established an AMQP connection with Hono.
-2. Client has established an AMQP link in role *sender* with Hono using target address `control-reply/${tenant_id}/{$device_id}`. Where `${device_id}` is an ID of the
+1. *Protocol Adapter* and *Business Application* have established an AMQP connection with the AMQP 1.0 Network.
+2. *Protocol Adatper* has established an AMQP link in role *sender* with Hono using target address `control-reply/${tenant_id}/{$device_id}`. Where `${device_id}` is an ID of the
 device that received the control message and `${tenant_id}` is the ID of the tenant the device belongs to.
+3. *Business Application* has established an AMQP link in role *receiver* with the AMQP 1.0 Network. 
 
-**Message Flow**
-
-*TODO* add sequence diagram
+(The message flow is shown in the sequence diagram above to show the whole sequence in one place)
 
 **Message Format**
 
-The following table provides an overview of the properties a client needs to set on a *Control* message.
+The following table provides an overview of the properties the *Protocol Adapter* needs to set on a *Control-Reply* message.
```

Comment:
IMHO this should be the spec of the response message an application gets in return to sending a command.